src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Greg0ire\Enum\Bridge\Symfony\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * @author Marcin Klimek <marcin.r.k@o2.pl>
+ */
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('greg0ire_enum');
+
+        $rootNode
+            ->children()
+                ->booleanNode('use_translator')->defaultTrue()->end()
```

Comment:
Please add a call to `info()` here, with a description of this configuration node